src/styles/item-tile-styles.ts

```diff
@@ -12,16 +12,13 @@ export const baseItemTileStyles = css`
     flex-direction: column;
```

Comment:
blocker: I'm not sure if we have a coding standard for whitespace in .css definitions. But we do want whitespace between each selector and rule set. Please undo this change.

Again, `yarn format` should do this, but I'm not sure. Please `yarn format` before every commit in this project.

Reply:
Not resolved.

Please move file to `src/tiles/grid/` next to `item-tile.ts`